Auto update failing for sfml

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [sfml](https://github.com/termux/termux-packages/blob/676ce7d5d6cd0546ca3b5c83de622be7895522a7/x11-packages/sfml) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating sfml [Current version: 3.0.2]
INFO: package being updated to 3.1.0.
Downloading https://github.com/SFML/SFML/archive/refs/tags/3.1.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  988k    0  988k    0     0   885k      0 --:--:--  0:00:01 --:--:--  885k
100 6206k    0 6206k    0     0  3945k      0 --:--:--  0:00:01 --:--:-- 11.1M

You are about to commit these changes:

--------------------
diff --git a/x11-packages/sfml/build.sh b/x11-packages/sfml/build.sh
index 2b35e055fb..83a29f6832 100644
--- a/x11-packages/sfml/build.sh
+++ b/x11-packages/sfml/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://www.sfml-dev.org/
 TERMUX_PKG_DESCRIPTION="A simple, fast, cross-platform and object-oriented multimedia API"
 TERMUX_PKG_LICENSE="ZLIB"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="3.0.2"
+TERMUX_PKG_VERSION="3.1.0"
 TERMUX_PKG_SRCURL=https://github.com/SFML/SFML/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=0034e05f95509e5d3fb81b1625713e06da7b068f210288ce3fd67106f8f46995
+TERMUX_PKG_SHA256=91209a112c2bd0bc6f4ce0d5f3e413cfb48b57c0de59f5507dc81f71b1ad7a5c
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="freetype, libc++, libflac, libogg, libvorbis, libx11, libxcursor, libxi, libxrandr, openal-soft, opengl"
 # -DBUILD_SHARED_LIBS=ON is required to prevent this error when importing SFML using CMake:
--------------------

bump(x11/sfml): 3.1.0


Not committing to Git!

INFO: Trying to build package.
INFO: Package sfml exists in x11 repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
Loading restricted AppArmor profile...
termux - building sfml for arch aarch64...
::group::INFO: Fetching repo metadata
[packages-cf.termux.dev-apt-termux-main]: Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
[packages-cf.termux.dev-apt-termux-main]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-main]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-main]: 
[packages-cf.termux.dev-apt-termux-main]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]: 100 13134  100 13134    0     0  50948      0 --:--:-- --:--:-- --:--:-- 51105
[packages-cf.termux.dev-apt-termux-main]: Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
[packages-cf.termux.dev-apt-termux-main]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-main]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-root]: Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
[packages-cf.termux.dev-apt-termux-root]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-root]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-x11]: Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
[packages-cf.termux.dev-apt-termux-x11]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-x11]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-root]: 
[packages-cf.termux.dev-apt-termux-root]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-root]: 100 13320  100 13320    0     0  17301      0 --:--:-- --:--:-- --:--:-- 17298
[packages-cf.termux.dev-apt-termux-root]: 100 13320  100 13320    0     0  17298      0 --:--:-- --:--:-- --:--:-- 17298
[packages-cf.termux.dev-apt-termux-root]: Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
[packages-cf.termux.dev-apt-termux-root]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-root]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-x11]: 
[packages-cf.termux.dev-apt-termux-x11]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-x11]:  83 13120   83 10934    0     0  13034      0  0:00:01 --:--:--  0:00:01 13032
[packages-cf.termux.dev-apt-termux-x11]: 100 13120  100 13120    0     0  15637      0 --:--:-- --:--:-- --:--:-- 15637
[packages-cf.termux.dev-apt-termux-x11]: Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
[packages-cf.termux.dev-apt-termux-x11]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[packages-cf.termux.dev-apt-termux-x11]:                                  Dload  Upload   Total   Spent    Left  Speed
[packages-cf.termux.dev-apt-termux-main]: 
[packages-cf.termux.dev-apt-termux-main]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]:   0   833    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-main]: 100   833  100   833    0     0   1055      0 --:--:-- --:--:-- --:--:--  1054
[packages-cf.termux.dev-apt-termux-main]: gpg: Signature made Thu 16 Apr 2026 07:05:14 PM UTC
[packages-cf.termux.dev-apt-termux-main]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-main]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-main]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
[packages-cf.termux.dev-apt-termux-root]: 
[packages-cf.termux.dev-apt-termux-root]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-root]: 100   833  100   833    0     0   1097      0 --:--:-- --:--:-- --:--:--  1098
[packages-cf.termux.dev-apt-termux-root]: gpg: Signature made Wed 15 Apr 2026 06:57:25 PM UTC
[packages-cf.termux.dev-apt-termux-root]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-root]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-root]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
[packages-cf.termux.dev-apt-termux-x11]: 
[packages-cf.termux.dev-apt-termux-x11]:   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[packages-cf.termux.dev-apt-termux-x11]: 100   833  100   833    0     0   1060      0 --:--:-- --:--:-- --:--:--  1061
[packages-cf.termux.dev-apt-termux-x11]: gpg: Signature made Thu 16 Apr 2026 04:20:38 PM UTC
[packages-cf.termux.dev-apt-termux-x11]: gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
[packages-cf.termux.dev-apt-termux-x11]: gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
[packages-cf.termux.dev-apt-termux-x11]: gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
::endgroup::
Downloading dependency brotli@1.2.0 if necessary...
Found brotli in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting brotli to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libbz2@1.0.8-8 if necessary...
Found libbz2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libbz2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bzip2@1.0.8-8 if necessary...
Found bzip2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting bzip2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency zlib@1.3.2 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libpng@1.6.58 if necessary...
Found libpng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libpng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency freetype@2.14.3 if necessary...
Found freetype in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting freetype to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libc++@29 if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libogg@1.3.6-1 if necessary...
Found libogg in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libo/libogg/libogg_1.3.6-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  184k  100  184k    0     0  2227k      0 --:--:-- --:--:-- --:--:-- 2246k
extracting libogg to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libflac@1.5.0-1 if necessary...
Found libflac in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libf/libflac/libflac_1.5.0-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  198k  100  198k    0     0  2260k      0 --:--:-- --:--:-- --:--:-- 2276k
extracting libflac to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency flac@1.5.0-1 if necessary...
Found flac in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/f/flac/flac_1.5.0-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  103k  100  103k    0     0  1394k      0 --:--:-- --:--:-- --:--:-- 1412k
extracting flac to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libvorbis@1.3.7-4 if necessary...
Found libvorbis in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libv/libvorbis/libvorbis_1.3.7-4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  255k  100  255k    0     0  2335k      0 --:--:-- --:--:-- --:--:-- 2345k
extracting libvorbis to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-support@29-1 if necessary...
Found libandroid-support in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-support to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xorg-util-macros@1.20.2 if necessary...
Found xorg-util-macros in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorg-util-macros to /home/builder/.termux-build/_cache-all...
Downloading dependency xorgproto@2025.1 if necessary...
Found xorgproto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorgproto to /home/builder/.termux-build/_cache-all...
Downloading dependency libxau@1.0.12-2 if necessary...
Found libxau in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxau to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxdmcp@1.1.5-2 if necessary...
Found libxdmcp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxdmcp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xcb-proto@1.17.0-2 if necessary...
Found xcb-proto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xcb-proto to /home/builder/.termux-build/_cache-all...
Downloading dependency python-xcbgen@1.17.0-2 if necessary...
Found python-xcbgen in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-xcbgen to /home/builder/.termux-build/_cache-all...
Downloading dependency libxcb@1.17.0-1 if necessary...
Found libxcb in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxcb to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xtrans@1.6.0 if necessary...
Found xtrans in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xtrans to /home/builder/.termux-build/_cache-all...
Downloading dependency libx11@1.8.13 if necessary...
Found libx11 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libx11 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxfixes@6.0.2 if necessary...
Found libxfixes in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxfixes/libxfixes_6.0.2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 11504  100 11504    0     0   156k      0 --:--:-- --:--:-- --:--:--  158k
extracting libxfixes to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxrender@0.9.12-1 if necessary...
Found libxrender in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxrender to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxcursor@1.2.3-1 if necessary...
Found libxcursor in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxcursor/libxcursor_1.2.3-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 27452  100 27452    0     0   452k      0 --:--:-- --:--:-- --:--:--  454k
extracting libxcursor to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxext@1.3.7 if necessary...
Found libxext in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxext to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxi@1.8.2-1 if necessary...
Found libxi in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxi/libxi_1.8.2-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  138k  100  138k    0     0  1690k      0 --:--:-- --:--:-- --:--:-- 1705k
extracting libxi to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxrandr@1.5.5 if necessary...
Found libxrandr in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxrandr/libxrandr_1.5.5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 22264  100 22264    0     0   265k      0 --:--:-- --:--:-- --:--:--  268k
extracting libxrandr to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openal-soft@1.25.1 if necessary...
Found openal-soft in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/o/openal-soft/openal-soft_1.25.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  630k  100  630k    0     0  5793k      0 --:--:-- --:--:-- --:--:-- 5836k
extracting openal-soft to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libglvnd@1.7.0-1 if necessary...
Found libglvnd in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libg/libglvnd/libglvnd_1.7.0-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  174k  100  174k    0     0  1826k      0 --:--:-- --:--:-- --:--:-- 1833k
extracting libglvnd to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libglvnd-dev@1.7.0-1 if necessary...
Found libglvnd-dev in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libg/libglvnd-dev/libglvnd-dev_1.7.0-1_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  146k  100  146k    0     0  1879k      0 --:--:-- --:--:-- --:--:-- 1904k
extracting libglvnd-dev to /home/builder/.termux-build/_cache-all...
Downloading dependency opengl@4.5 if necessary...
Found opengl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/o/opengl/opengl_4.5_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   692  100   692    0     0  10948      0 --:--:-- --:--:-- --:--:-- 10984
extracting opengl to /home/builder/.termux-build/_cache-all...
Downloading sfml@3.1.0 source from 'https://github.com/SFML/SFML/archive/refs/tags/3.1.0.tar.gz' if necessary...
Downloading https://github.com/SFML/SFML/archive/refs/tags/3.1.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 2562k    0 2562k    0     0  2145k      0 --:--:--  0:00:01 --:--:-- 2145k
100 6206k    0 6206k    0     0  4411k      0 --:--:--  0:00:01 --:--:-- 16.7M
Applying patch: cmake-Config.cmake.patch
Applying patch: include-SFML-Config.hpp.patch
Applying patch: src-SFML-Window-CMakeLists.txt.patch
CMake Warning at /home/builder/.termux-build/_cache/cmake-4.3.1/share/cmake-4.3/Modules/Platform/Android-Determine.cmake:207 (message):
  Android: Did not detect API level from

    /home/builder/.termux-build/_cache/android-r29-api-24-v4/bin/clang

  or

    /home/builder/.termux-build/_cache/android-r29-api-24-v4/sysroot/usr/include/android/api-level.h

Call Stack (most recent call first):
  /home/builder/.termux-build/_cache/cmake-4.3.1/share/cmake-4.3/Modules/CMakeDetermineSystem.cmake:191 (include)
  CMakeLists.txt:28 (project)


-- Android: Targeting API '24' with architecture 'arm64', ABI 'arm64-v8a', and processor 'aarch64'
-- The C compiler identification is Clang 21.0.0
-- The CXX compiler identification is Clang 21.0.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /home/builder/.termux-build/_cache/android-r29-api-24-v4/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /home/builder/.termux-build/_cache/android-r29-api-24-v4/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE
-- Found X11: /data/data/com.termux/files/usr/include  found components: Xrandr Xcursor Xi
-- Looking for XOpenDisplay in /data/data/com.termux/files/usr/lib/libX11.so;/data/data/com.termux/files/usr/lib/libXext.so
-- Looking for XOpenDisplay in /data/data/com.termux/files/usr/lib/libX11.so;/data/data/com.termux/files/usr/lib/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - not found
-- Looking for shmat in ipc
-- Looking for shmat in ipc - not found
-- Found OpenGL: /data/data/com.termux/files/usr/lib/libOpenGL.so  found components: OpenGL
CMake Error at /home/builder/.termux-build/_cache/cmake-4.3.1/share/cmake-4.3/Modules/FindPackageHandleStandardArgs.cmake:290 (message):
  Could NOT find MbedTLS (missing: MBEDTLS_INCLUDE_DIR MBEDTLS_LIBRARY
  MBEDX509_LIBRARY MBEDCRYPTO_LIBRARY)
Call Stack (most recent call first):
  /home/builder/.termux-build/_cache/cmake-4.3.1/share/cmake-4.3/Modules/FindPackageHandleStandardArgs.cmake:654 (_FPHSA_FAILURE_MESSAGE)
  cmake/Modules/FindMbedTLS.cmake:94 (find_package_handle_standard_args)
  src/SFML/Network/CMakeLists.txt:60 (find_package)


-- Configuring incomplete, errors occurred!
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occurred when I last tried to update at 2026-04-16 19:10:46 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/24526954603">24526954603</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/mbedtls/build.sh
+++ b/packages/mbedtls/build.sh
@@ -4,7 +4,7 @@ TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_SRCURL=git+https://github.com/Mbed-TLS/mbedtls
 TERMUX_PKG_VERSION="3.6.4"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_REVISION=2
 TERMUX_PKG_BREAKS="mbedtls-dev"
 TERMUX_PKG_REPLACES="mbedtls-dev"
 TERMUX_PKG_AUTO_UPDATE=true
@@ -36,4 +36,8 @@ termux_step_pre_configure() {
 	if [ "$TERMUX_ARCH" = "i686" ]; then
 		scripts/config.py unset MBEDTLS_AESNI_C
 	fi
+
+	# for sfml
+	scripts/config.py set MBEDTLS_THREADING_C
+	scripts/config.py set MBEDTLS_THREADING_PTHREAD
 }

--- a/x11-packages/sfml/build.sh
+++ b/x11-packages/sfml/build.sh
@@ -2,11 +2,11 @@ TERMUX_PKG_HOMEPAGE=https://www.sfml-dev.org/
 TERMUX_PKG_DESCRIPTION="A simple, fast, cross-platform and object-oriented multimedia API"
 TERMUX_PKG_LICENSE="ZLIB"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="3.0.2"
+TERMUX_PKG_VERSION="3.1.0"
 TERMUX_PKG_SRCURL=https://github.com/SFML/SFML/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=0034e05f95509e5d3fb81b1625713e06da7b068f210288ce3fd67106f8f46995
+TERMUX_PKG_SHA256=91209a112c2bd0bc6f4ce0d5f3e413cfb48b57c0de59f5507dc81f71b1ad7a5c
 TERMUX_PKG_AUTO_UPDATE=true
-TERMUX_PKG_DEPENDS="freetype, libc++, libflac, libogg, libvorbis, libx11, libxcursor, libxi, libxrandr, openal-soft, opengl"
+TERMUX_PKG_DEPENDS="freetype, harfbuzz, libc++, libflac, libogg, libresolv-wrapper, libssh2, libvorbis, libx11, libxcursor, libxi, libxrandr, mbedtls, openal-soft, opengl"
 # -DBUILD_SHARED_LIBS=ON is required to prevent this error when importing SFML using CMake:
 # Requested SFML configuration (Shared) was not found
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="

--- a/x11-packages/sfml/src-SFML-Network-CMakeLists.txt.patch
+++ b/x11-packages/sfml/src-SFML-Network-CMakeLists.txt.patch
@@ -1,0 +1,11 @@
+--- a/src/SFML/Network/CMakeLists.txt
++++ b/src/SFML/Network/CMakeLists.txt
+@@ -53,7 +53,7 @@
+ if(SFML_OS_WINDOWS)
+     target_link_libraries(sfml-network PRIVATE Crypt32 ws2_32 Dnsapi)
+ elseif(SFML_OS_LINUX OR SFML_OS_IOS OR SFML_OS_MACOS)
+-    target_link_libraries(sfml-network PRIVATE resolv)
++    target_link_libraries(sfml-network PRIVATE resolv_wrapper)
+ endif()
+ 
+ if(SFML_USE_SYSTEM_DEPS)

